src/components/Picker/Picker.visual.tsx

```diff
@@ -13,5 +13,11 @@ visualize('Picker', () => {
     snap('default', () => <Picker {...defaultProps} />);
     snap('prev disabled', () => <Picker prevDisabled {...defaultProps} />);
     snap('next disabled', () => <Picker nextDisabled {...defaultProps} />);
+    snap('next aria label', () => (
+      <Picker nextAriaLabel="next" {...defaultProps} />
+    ));
+    snap('prev aria label', () => (
+      <Picker prevAriaLabel="prev" {...defaultProps} />
+    ));
```

Comment:
These aren't needed actually, cause they don't change anything visually.
If anything I would add these props to all the other visual tests by default